strawberry/schema/schema.py

```diff
@@ -66,15 +73,15 @@ def __init__(
             formatted_errors = "\n\n".join(f"❌ {error.message}" for error in errors)
             raise ValueError(f"Invalid Schema. Errors:\n\n{formatted_errors}")
 
-        self.query = self.type_map[query_type.name]
+        self.query = self.schema_converter.type_map[query_type.name]
 
     def get_type_by_name(
         self, name: str
     ) -> Optional[
         Union[TypeDefinition, ScalarDefinition, EnumDefinition, StrawberryUnion]
     ]:
-        if name in self.type_map:
-            return self.type_map[name].definition
+        if name in self.schema_converter.type_map:
```

Comment:
It was mainly done to avoid passing the map around all the time. Do you think that the Schema and the schema converter should share a reference to the same type map?

Reply:
uhm, no, I think we can add a property to the schema if we need the type map in future, was just curious 😊